src/webfaf/webfaf_main.py

```diff
@@ -131,9 +131,9 @@ def about():
 def component_names_json():
     sub = db.session.query(distinct(Report.component_id)).subquery()
     comps = (db.session.query(OpSysComponent.name)
-                       .filter(OpSysComponent.id.in_(sub))
-                       .distinct(OpSysComponent.name)
-                       .all())
+             .filter(OpSysComponent.id.in_(sub))          
+             .distinct(OpSysComponent.name)               
+             .all())                                      
```

Comment:
I am not sure if I am correct, but GitHub is showing many trailing white spaces, is GitHub right?

Reply:
Yes. You are right.